salt/modules/napalm_mod.py

```diff
@@ -1286,3 +1293,156 @@ def rpc(command, **kwargs):
     napalm_map.update(default_map)
     fun = napalm_map.get(__grains__['os'], 'napalm.netmiko_commands')
     return __salt__[fun](command, **kwargs)
+
+
+@depends(HAS_CISCOCONFPARSE)
+def config_find_lines(regex, source='running'):
+    r'''
+    .. versionadded:: Fluorine
+
+    Return the configuration lines that match the regular expressions from the
+    ``regex`` argument. The configuration is read from the network device
+    interrogated.
+
+    regex
+        The regular expression to match the configuration lines against.
+
+    source: ``running``
+        The configuration type to retrieve from the network device. Default:
+        ``running``. Available options: ``running``, ``startup``, ``candidate``.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' napalm.config_find_lines '^interface Ethernet1\d'
+    '''
+    config_txt = __salt__['net.config'](source=source)['out'][source]
+    return __salt__['ciscoconfparse.find_lines'](config=config_txt,
+                                                 regex=regex)
+
+
+@depends(HAS_CISCOCONFPARSE)
+def config_lines_w_child(parent_regex, child_regex, source='running'):
```

Comment:
I think the functions in this file could use some new names as well, just like above. I won't comment on each one though, you get the idea.

Reply:
Please see https://github.com/saltstack/salt/pull/48733#discussion_r206498279 for my reasoning - happy to update if you think that's best - no problem at all!